### PR TITLE
docs: add `editPageLink` in demo

### PR DESCRIPTION
**Pull request by Charan167**
https://github.com/user-attachments/assets/1e319a27-01f4-42eb-b599-c9a3f5fdb2fd

- Added editPageLink to demo lessons

### CODE REVIEW

**Review by AriPerkkio — CHANGES_REQUESTED** [review]
Great work @Charan167!

While this already works really nicely, we can reduce the amount of code by defining it only in top-level metadata, here:

https://github.com/stackblitz/tutorialkit/blob/c8ce1826aeb79ef878a4e5e7104369ce900eb76d/docs/demo/src/content/tutorial/meta.md?plain=1#L1-L6

So looking at your code, I think something like this could work there:

```yaml
---
editPageLink: https://github.com/stackblitz/tutorialkit/blob/main/docs/demo/src/content/tutorial/${path}?plain=1
---
```

The `${path}` here is dynamic parameter that TutorialKit will automatically apply there per-lesson.

**Comment by Charan167** [comment]
@AriPerkkio Forgive my noobness 😅. Please let me know if any more changes are needed. Also, feel free to let me know if you need help with any other issues—I'd be more than happy to work on them.

File: docs/demo/src/content/tutorial/meta.md [hunk 1]
```diff
@@ -2,5 +2,7 @@
 type: tutorial
 mainCommand: ['npm run dev', 'Starting http server']
 prepareCommands:
-  - ['npm install', 'Installing dependencies']
+- ['npm install', 'Installing dependencies']
+editPageLink: https://github.com/stackblitz/tutorialkit/blob/main/docs/demo/src/content/tutorial/${path}?plain=1
+
```
Comment (AriPerkkio) [inline docs/demo/src/content/tutorial/meta.md:7]:
```suggestion
```

File: docs/demo/src/content/tutorial/1-forms-css/3-fieldset/1-fieldset-element/content.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 type: lesson
 title: Fieldset element
 slug: fieldset-element
-focus: /index.html
+focus: /index.html
```
Comment (AriPerkkio) [inline docs/demo/src/content/tutorial/1-forms-css/3-fieldset/1-fieldset-element/content.md:5]:
```suggestion
focus: /index.html
```

**Review by AriPerkkio — CHANGES_REQUESTED** [review]

**Review by AriPerkkio — APPROVED** [review]
Looks good, thanks @Charan167!

> Also, feel free to let me know if you need help with any other issues—I'd be more than happy to work on them.

Awesome! Maybe these issues could be good next?

- https://github.com/stackblitz/tutorialkit/issues/300
- https://github.com/stackblitz/tutorialkit/issues/331

**Pull request merged**